### ISSUE TITLE
Amplify fails to deploy

### ISSUE THREAD

**Issue by dgwalters** [awslabs/sandbox-accounts-for-events#8]
[log.txt](https://github.com/awslabs/sandbox-accounts-for-events/files/9989822/log.txt)

**Comment by tmatid** [comment]
I'm getting the same error, have you been able to resolve this.

**Comment by tmatid** [comment]
What doesn't make sense is that 2 weeks ago the same code worked but when I try it again tonight I get an error

**Comment by moellr** [comment]
I was able to reproduce the issue. The specified amplify cli version suddenly breaks the Amplify deployment.
I have just pushed the changes to use a newer amplify cli version, should work now.

**Comment by tmatid** [comment]
Thank you very I had already resolved it and was going to share the changes I have made.I’ll be be active on this repo to watch out for any changes Sent from my iPhoneOn 22 Nov 2022, at 17:43, moellr ***@***.***> wrote:﻿
I was able to reproduce the issue. The specified amplify cli version suddenly breaks the Amplify deployment.
I have just pushed the changes to use a newer amplify cli version, should work now.
